bot/admin/admin.py

```diff
@@ -39,7 +39,7 @@ async def echo(self, ctx: commands.Context, channel: Optional[discord.TextChanne
         """
         await (channel or ctx).send(text)
 
-    @commands.command(name='embed', hidden=True)
+    @commands.group(name='embed', hidden=True, invoke_without_command=True)
```

Comment:
Does the parsing of the argument now work as expected with `invoke_without_command=True`?

Reply:
Yes!

The **actual** problem before was that when the message was correctly parsed and both methods would be invoked. Since the signature of the first method 'channel, color, text' did not fit with 'embed json channel {...}' The group method was not able to parse the channel and color correctly and so it threw an exception, so the invocation of the subcommand method never happened. 
Now only the second method will even be invoked so it works perfetly fine.